Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -159,7 +159,7 @@
     }
 
     void SetAllowsFigureCalls(bool allows) {
-        m_allowsFigureCalls = allows;
+        m_allowsFigureCalls += (allows ? 1 : -1);
```

Comment:
Negative values can be considered failures, right? Should we update this method to return an HRESULT?